scripts/data/get_blocks.sh

```diff
@@ -4,22 +4,25 @@ set -o pipefail;
 
 get_single_block() {
     local block_hash=$1
-    ./scripts/data/get_block.sh "$block_hash"
```

Comment:
It's a good time to add installation instructions to the README :)
Can you please update the doc to include the section about build dependencies?